Algorithms and Deep Learning Models/Spoken Language Translator/README.MD

```diff
@@ -99,4 +99,5 @@ Before running the application, ensure you have the following libraries installe
 
 -   **Improved Language Detection**: Enhance the accuracy of language detection using advanced machine learning techniques.
 -   **Additional Features**: Add features such as text-to-speech conversion and language identification for multilingual texts.
--   **Performance Optimization**: Optimize the translation process for faster and more efficient translations.
\ No newline at end of file
+-   **Performance Optimization**: Optimize the translation process for faster and more efficient translations.
+5. **beneifts**
```

Comment:
@Aasthaa10 i can see only one word got changed here, 

have you raised any issue for this before? if yes let me know .